src/main/java/pl/chojnacki/lesson3/homework/RectangularTriangle.java

```diff
@@ -0,0 +1,7 @@
+package pl.chojnacki.lesson3.homework;
+
+public class RectangularTriangle {
+    public static boolean isTriangleRectangular(double sideA, double sideB, double sideC) {
+        return sideA * sideA + sideB * sideB == sideC * sideC && sideA > 0 && sideB > 0 && sideC > 0;
```

Comment:
trochę bym to uporządkował, bo teraz jak na to patrzę, to zastanawiam się co jest sprawdzane :)